include/aws/http/private/h2_connection.h

```diff
@@ -117,7 +122,10 @@ enum aws_h2_data_encode_status {
     AWS_H2_DATA_ENCODE_ONGOING_WINDOW_STALLED,
 };
 
+/* When window size is too small to fit the possible padding into it, we stop sending data and wait for WINDOW_UPDATE */
 #define AWS_H2_MIN_WINDOW_SIZE (256)
+/* Initial connection flow-control window with 65535 bytes (RFC 6.9.2) */
+#define AWS_H2_INITIAL_WINDOW_SIZE (65535)
```

Comment:
why did you add `AWS_H2_INITIAL_WINDOW_SIZE`? we already had `aws_h2_settings_initial[AWS_H2_SETTINGS_INITIAL_WINDOW_SIZE]`?

Reply:
I was trying to send the window_update when the window size is too small, so that value will be used much more frequently. And the implementation is changed now, which makes this seems useless now. I'll remove it. 😅